pkg/psmdb/service.go

```diff
@@ -63,7 +63,12 @@ func Service(cr *api.PerconaServerMongoDB, replset *api.ReplsetSpec) *corev1.Ser
 			}
 		}
 	}
-
+	if cr.CompareVersion("1.16.0") < 0 {
```

Comment:
we only want to do this for v1.16.0 and further
```suggestion
	if cr.CompareVersion("1.16.0") >= 0 {
```